pkg/controller/migmigration/migrate.go

```diff
@@ -22,19 +22,19 @@ import (
 	vrunner "github.com/fusor/mig-controller/pkg/velerorunner"
 )
 
-func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) error {
+func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (bool, error) {
```

Comment:
I might consider a named return value for `requeue bool` here. It doesn't feel obvious that true/false here refers to "requeue" rather than something like "success".